client/a8c-for-agencies/sections/marketplace/shopping-cart/shopping-cart-menu/item.tsx

```diff
@@ -19,8 +18,6 @@ export default function ShoppingCartMenuItem( { item, onRemoveItem }: ItemProps
 	const translate = useTranslate();
 	const userProducts = useSelector( getProductsList );
 
-	const siteId = getQueryArg( window.location.href, 'site_id' )?.toString();
```

Comment:
This is just fixing lint issues, from this [commit](https://github.com/Automattic/wp-calypso/blob/02d1c94499bca9f015e61624ac08813ad496c87f/client/a8c-for-agencies/sections/marketplace/shopping-cart/shopping-cart-menu/item.tsx)

`SiteId` was not being used then lint was not allowing me to merge/push